go.mod

```diff
@@ -42,7 +42,7 @@ require (
 	golang.org/x/oauth2 v0.0.0-20200107190931-bf48bf16ab8d // indirect
 	golang.org/x/sync v0.0.0-20200625203802-6e8e738ad208
 	golang.org/x/sys v0.0.0-20200625212154-ddb9806d33ae // indirect
-	golang.org/x/tools v0.0.0-20201013053347-2db1cd791039 // indirect
+	golang.org/x/tools v0.0.0-20201019175715-b894a3290fff // indirect
```

Comment:
Why this change is needed ?

Reply:
I ran make check, I think this go added that time. Will revert it.